python/tvm/relay/qnn/op/_qnn.py

```diff
@@ -85,12 +91,72 @@ def simulated_dequantize_compute(attrs, inputs, output_type):
 register_strategy("qnn.conv2d", strategy.qnn_conv2d_strategy)
 
 
+def _get_clip_dtype_bounds(dtype):
+    """Returns the minimum and maximum values of a C integer data type."""
+    assert "int" in dtype
+    bits = int(dtype[dtype.find("int") + 3 :])
+
+    if dtype.startswith("int"):
+        return (-(2 ** (bits - 1)), 2 ** (bits - 1) - 1)
```

Comment:
Just a nit comment... np.iinfo(dtype).min / np.iinfo(dtype).max are not suitable?

Reply:
That's much cleaner - fixed!